src/Book.ts

```diff
@@ -29,15 +35,40 @@ export class Book<U extends ITransaction = ITransaction, J extends IJournal = IJ
     if (typeof this.maxAccountPath !== "number" || !Number.isInteger(this.maxAccountPath) || this.maxAccountPath < 0) {
       throw new BookConstructorError("Invalid value for maxAccountPath provided.");
     }
+
+    if (typeof this.balanceSnapshotSec !== "number" || this.balanceSnapshotSec < 0) {
+      throw new BookConstructorError("Invalid value for balanceSnapshotSec provided.");
```

Comment:
This can be 0.001 - which is 1 millisecond. In unit test above we have 0.05. 

We can have `balanceSnapshotSec` in ms instead of sec if you believe it's better.

Reply:
Ah. Ok. Yeah maybe using integers and ms is better because what happens if somebody uses 0.00001 or as we use seconds thinks that seconds are only possible.

Reply:
I would also suggest using ms instead of seconds